packages/sdk-js/package.json

```diff
@@ -15,7 +15,7 @@
     "src"
   ],
   "engines": {
-    "node": ">=10"
```

Comment:
We don't want to update these SDK package.json files.  We want the smallest Node version possible here since that means the most people are able to use our library.